<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>796</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>스 타 벅 스 커 피</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>ku1.jpg</normaloff>ku1.jpg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>821</width>
      <height>621</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="toolTipDuration">
     <number>-1</number>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QTabBar:tab {
 height: 100px; width: 410px;
 }</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Drink</string>
     </attribute>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>131</width>
        <height>131</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>230</y>
        <width>71</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>추가</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>180</y>
        <width>131</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>아이스 아메리카노</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>200</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>4100 원</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2_2">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>230</y>
        <width>61</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>취소</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>30</y>
        <width>131</width>
        <height>131</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>180</y>
        <width>131</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>허니 자몽 블랙티</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>200</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>5800 원</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>230</y>
        <width>71</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>추가</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3_3">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>230</y>
        <width>61</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>취소</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>30</y>
        <width>131</width>
        <height>131</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>180</y>
        <width>171</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>제주 쑥떡 크림 프라푸치노</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>200</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>7500 원</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>230</y>
        <width>71</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>추가</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4_4">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>230</y>
        <width>61</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>취소</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>280</y>
        <width>131</width>
        <height>131</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>280</y>
        <width>131</width>
        <height>131</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>290</y>
        <width>131</width>
        <height>131</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>430</y>
        <width>131</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>돌체 콜드브루</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>450</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>5800 원</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>480</y>
        <width>71</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>추가</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5_5">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>480</y>
        <width>61</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>취소</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>430</y>
        <width>131</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>자바칩 프라푸치노</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>450</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>6100 원</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>480</y>
        <width>71</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>추가</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_6_6">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>480</y>
        <width>61</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>취소</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>430</y>
        <width>141</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>말차 크림 프라푸치노</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>450</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>6300 원</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_7">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>480</y>
        <width>71</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>추가</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_7_7">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>480</y>
        <width>61</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>취소</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Side</string>
     </attribute>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>131</width>
        <height>131</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>30</y>
        <width>131</width>
        <height>131</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_22">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>30</y>
        <width>131</width>
        <height>131</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_23">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>280</y>
        <width>131</width>
        <height>131</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_24">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>280</y>
        <width>131</width>
        <height>131</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_25">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>290</y>
        <width>131</width>
        <height>131</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_26">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>180</y>
        <width>131</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>프렌치 크루와상</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_27">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>180</y>
        <width>131</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>클래식 스콘</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_28">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>180</y>
        <width>141</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>멕시칸 라이스 브리또</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_29">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>430</y>
        <width>131</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>에그에그 샌드위치</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_30">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>430</y>
        <width>141</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>포크 커틀릿 샌드위치</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_31">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>430</y>
        <width>201</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>제주 흑돼지 아보카도 샌드위치</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_32">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>200</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>3200 원</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_33">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>200</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>3200 원</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_34">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>200</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>5600 원</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_35">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>450</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>4400 원</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_36">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>450</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>5900 원</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_37">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>450</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>6200 원</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_8">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>230</y>
        <width>71</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>추가</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_8_8">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>230</y>
        <width>61</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>취소</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_9">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>230</y>
        <width>71</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>추가</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_9_9">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>230</y>
        <width>61</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>취소</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_10">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>230</y>
        <width>71</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>추가</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_10_10">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>230</y>
        <width>61</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>취소</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_11">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>480</y>
        <width>71</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>추가</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_11_11">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>480</y>
        <width>61</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>취소</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_12">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>480</y>
        <width>71</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>추가</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_12_12">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>480</y>
        <width>61</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>취소</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_13">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>480</y>
        <width>71</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>추가</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_13_13">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>480</y>
        <width>61</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>취소</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>630</y>
      <width>241</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>630</y>
      <width>401</width>
      <height>141</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>630</y>
      <width>181</width>
      <height>141</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>결제</string>
    </property>
   </widget>
   <widget class="QDialog" name="kdialog">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>109</y>
      <width>491</width>
      <height>471</height>
     </rect>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="whatsThis">
     <string/>
    </property>
    <property name="accessibleName">
     <string/>
    </property>
    <property name="modal">
     <bool>false</bool>
    </property>
    <widget class="QTextBrowser" name="textBrowser_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>321</width>
       <height>471</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>50</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_38">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>10</y>
       <width>131</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>돈을 투입하세요</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_14">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>80</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>확인</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_15">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>150</y>
       <width>151</width>
       <height>121</height>
      </rect>
     </property>
     <property name="text">
      <string>현금 결제</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_16">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>340</y>
       <width>151</width>
       <height>121</height>
      </rect>
     </property>
     <property name="text">
      <string>카드 결제</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <slot>slot1()</slot>
  <slot>slot2()</slot>
  <slot>slot4()</slot>
  <slot>slot3()</slot>
 </slots>
</ui>
